website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,260 @@ aliases:
+the steering committee for decision. Common ways to do this are via an email
+alias for the Steering Committee, filing an issue in a specific repository, or
+posting in a Steering Committee Slack channel.
+
+### Composition
+
+This section requires some extra work and addition thought put into how you plan
+to handle election terms and replacing Steering Committee members whose terms
+are up for election.
+
+We recommend using staggered 2-year terms where only half of the seats are up
+for election in any given year to maintain continuity. This keeps enough people
+on the Steering Committee to provide context and onboarding for new members.
+
+If you are starting from scratch with a bootstrapped committee, and need a
+transition period, but plan to have staggered 2-year terms, we recommend that for
+the first year of the Steering Committee half of the seats should be elected for a
+1-year term, and the other half should be elected for a 2-year term.
+
+While not recommended you can elect-everyone-at-once and have all of the members
+elected at the same time to serve for a specified term.
+
+### Election Procedure
+
+Note: CNCF will soon have a better, GitOps-driven, online election tool
+available.  At that time, projects will want to revise portions of this
+procedure.
+
+The template contains recommended text for timelines, election officers, and
+eligibiity to vote. These sections are all relatively straightforward, and we
+recommend using the templated text, but you will need to determine the threshold
+for voter eligibility and specify where community members can find the list of
+eligible voters.
+
+**Candidate Eligibility**
+
+You will need to determine what criteria a person needs to meet in order to be
+eligible to run as a candidate in the election. Some common criteria include:
+being in a project member role in the [Contributor Ladder], listed as an
+approver in one or more OWNERS files, or similar.
+
+**Voting Procedure**
+
+For most elections, we recommend using 
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+[CIVS](http://civs.cs.cornell.edu/) using the IRV method as specified in the
+template.
+
+**Limitations on Company Representation**
+
+We recommend placing limits on company representation, which ensures a more fair
+and neutral decision-making body that isn't over-represented by a single vendor.
+Exactly where you set those limits depends on your current community
+composition, the number of seats on your Steering Committee, overall employer
+representation within your community, and other factors. We strongly recommend
+discussing this with [TAG Contributor Strategy] before deciding how you plan to
+limit company representation.
+
+## Vacancies
+
+You will need a process for handling vacancies, and we recommend using the text
+currently found in the template.
+
+### Changes to the charter
+
+Changes to your charter should refer back to the "Decision Process" section and
```

Comment:
I think we decided in today's governance meeting that "Decision Process" should use the lazy consensus link that is at the bottom of this page? Otherwise it's unused.

Reply:
Done!